docs/theory/Validation_pipe_diameter_sizing.rst

```diff
@@ -0,0 +1,1148 @@
+    +==============================+==========================+==========================+==========================+==========================+==========================+==========================+
+    | T_demand                     | 77.17                    |  77.80                   | 78.20                    | 79.63                    | 79.72                    | 79.78                    |
+    +------------------------------+--------------------------+--------------------------+--------------------------+--------------------------+--------------------------+--------------------------+
+    | Heat loss tot [W]            | 1051803                  | 811278                   | 659814                   | 1144002                  | 867882                   |  699118                  |
+    +------------------------------+--------------------------+--------------------------+--------------------------+--------------------------+--------------------------+--------------------------+
+    | Heat loss supply [W]         | 762207                   | 583137                   | 471940                   | 836270                   | 627677                   |   502494                 |
+    +------------------------------+--------------------------+--------------------------+--------------------------+--------------------------+--------------------------+--------------------------+
+    | Heat loss return [W]         | 289598                   | 228140                   | 187874                   | 307732                   | 240204                   |  196623                  |    
+    +------------------------------+--------------------------+--------------------------+--------------------------+--------------------------+--------------------------+--------------------------+
+    |:math:`C_{var,opex,year}` [M€]| 1936                     | 1894                     | 1868                     | 15968                    | 15920                    | 15890                    |
+    +------------------------------+--------------------------+--------------------------+--------------------------+--------------------------+--------------------------+--------------------------+
+    |:math:`C_{var,opex,hour}` [M€]| 0.221                    | 0.216                    | 0.213                    | 1.823                    | 1.817                    | 1.814                    |
+    +------------------------------+--------------------------+--------------------------+--------------------------+--------------------------+--------------------------+--------------------------+
+    
+The results for 10 MW show a more dominant drop in temperature of the 25 km long pipes compared to those of 2.5 km. 
+Because of this it does make more sense to insulate the pipes with a higher class. 
```

Comment:
with a "higher length"?

Reply:
Changed it to: "Because of this it does make more sense to insulate longer pipes with a higher class. "

Reply:
Indeed, it;s more clear now.